### ISSUE TITLE
Regarding Speakers.pth file

### ISSUE THREAD

**Issue by sachin7695** [AI4Bharat/Indic-TTS#25]
what to do if i want to use a different speakers.pth file of fastpitch (fine tuned on hindi language ) can someone help me to know about speakers.pth file? I just want to use a celebrity voice on the systhesis part so what should i do to solve this isssue? i thought that i should fine tune coqui tts on a celebrity voice then use that as a speakers.pth to cater a celebrity voice in indic tts but i encountered error? i want help
![image](https://github.com/user-attachments/assets/5e633e49-95b5-43df-82d0-aeac6ceefcff)

**Comment by VivekGangurde94** [comment]
we dont need speaker.pth file for fine tunning.

**Comment by sachin7695** [comment]
yes i understood thank you .. can you provide any script to follow that for fine tuning?

**Comment by VivekGangurde94** [comment]
for finetunning you have to understand what type of data should be provide and which format for muti speaker we used vctk dataset for and for single speaker we have used ljspeech dataset format u can fine tune coqui tts.

**Comment by RamakrishnaChaitanya** [comment]
Hi @sachin7695, have you resolved this issue? I'm also struck at this point. Is there a way to derive speaker embeddings using Coqui-ai and use them instead while generating the speech?

**Comment by RamakrishnaChaitanya** [comment]
@sachin7695 I too faced the similar issue. It looks like you loaded the wrong checkpoint or config file. Please check. I was able to resolve this issue after loading the correct checkpoint.
